I added my name

Nics-Github opened the pull request:
Maybe

Commit:
I put my name

Commit:
Third attempt

Commit:
Trying again

Commit:
I think I'm doing this correctly now.

beanumber reviewed (changes requested):
@Nics-Github thank for your PR. 

We don't want you to commit the changes to the .Rproj file. Instead can you create a new PR with just the changes to README.md?

Commit:
Merge branch 'main' of https://github.com/Nics-Github/facdev22-demo into main

Commit:
Fifth Attempt, am I doing this correctly yet?

Merge branch 'main' of https://github.com/DSC-WAV/facdev22-demo into main

# Conflicts:
#	README.md

Commit:
Sixth Attempt

Commit:
MAybe

beanumber commented on README.md:
```diff
@@ -9,4 +9,5 @@ The list of participants in this demo are:
-- Eric Carreira
\ No newline at end of file
+- Eric Carreira
+- Nic Schwab!!
```
@Nics-Github can you please remove the exclamation points?
